docs/selectively-triggering-aktualizr.adoc

```diff
@@ -58,3 +58,20 @@ Installation reports are sent when aktualizr polls the server for updates, so th
 
     aktualizr --running-mode=check
 
+=== Check for campaigns target the device
+
+Campaigns provide a way to target software updates to specific groups of devices. You can set up campaigns in the OTA Connect web app. When a device comes online that matches the targeting criteria of a campaign, the OTA Connect server notifies the device that new software updates are available. 
+
+    aktualizr --running-mode=campaign_check
+
+If the device is targeted by one or more campaigns, aktualizr prints details of each applicable campaign including the campaign ID. You can then use the campaign ID for the `campaign_accept` command.
+
+=== Accept campaigns that target the device
+
+This running mode works together with the `campaign_check` mode. After you have received a list of applicable campaigns, you can accept the campaigns so that they are included in the software update process.
+
+    aktualizr --running-mode=campaign_accept --campaign-id=<CAMPAIGN_ID>
+
+For example:
+
+    aktualizr --running-mode=campaign_accept --campaign-id=4df1a199-59d4-47f6-a261-d79b83020f65
```

Comment:
I would add that the update scheduled by the campaign will then become available to the device (`check` running mode or regular full update cycle)

Reply:
Ok done, see latest commit